Cargo.toml

```diff
@@ -71,3 +71,15 @@ path = "crates/fj-viewer"
 [workspace.dependencies.fj-window]
 version = "0.47.0"
 path = "crates/fj-window"
+
+[profile.dev.package.'*']
+opt-level = 3
```

Comment:
I'm very doubtful that this provides a benefit. I think it will increase the compile times of everything in `crates/`, whenever you run an example model (which is one of the main forms of testing when developing in this repository).

Reply:
Ah, thanks for clarifying. I must have thought it it applied to dependencies in general, regardless of whether they're in the workspace or not. That's much better.



> I think I added this to help with `egui`?

It's worth noting then, that `egui` is no longer a dependency.